Fedoraware/Fedoraware-TF2/src/Features/Menu/Menu.cpp

```diff
@@ -2056,6 +2056,7 @@ void CMenu::SettingsWindow()
 		{
 			if (ColorPicker("Menu accent", Vars::Menu::Colors::MenuAccent.Value)) { LoadStyle(); } SameLine(); Text("Menu accent");
 			if (Checkbox("Alternative Design", &Vars::Menu::ModernDesign.Value)) { LoadStyle(); }
+			if (Checkbox("Draw Snow", &Vars::Menu::DrawSnow.Value)) { LoadStyle(); }
```

Comment:
I don't want people to disable snow 😠

Reply:
Actually, as long as it still says "Merry Christmas" it's fine. But can you make the snow enabled by default?

Reply:
> Actually, as long as it still says "Merry Christmas" it's fine. But can you make the snow enabled by default?

maybe make rain by default? its prettier and lags less (200 raindrops instead of 1000 snowflakes)

Reply:
How about you combine the two things into one "Enable Weather" option and make it switch between snow and rain depending on the month?
Also feel free to reduce the amount of snowflakes, if it lags too much.

Reply:
> How about you combine the two things into one "Enable Weather" option and make it switch between snow and rain depending on the month? Also feel free to reduce the amount of snowflakes, if it lags too much.



> How about you combine the two things into one "Enable Weather" option and make it switch between snow and rain depending on the month? Also feel free to reduce the amount of snowflakes, if it lags too much.

i could make it rain in autumn and snow in winter

Reply:
> How about you combine the two things into one "Enable Weather" option and make it switch between snow and rain depending on the month? Also feel free to reduce the amount of snowflakes, if it lags too much.

Added.